libbeat/ml-importer/importer.go

```diff
@@ -228,32 +228,26 @@ func checkResponse(r []byte) error {
 	}
 	for _, dashboard := range resp.Kibana.Dashboard {
 		if !dashboard.Success {
-			if dashboard.Exists {
+			if dashboard.Exists || strings.Contains(dashboard.Error.Message, "version conflict, document already exists") {
 				logp.Debug("machine-learning", "Dashboard already exists: %s", dashboard.ID)
```

Comment:
Can we add to this debug logging also `dashboard.Error.Message`? This could give some more context when debugging.